App/Redux/ProcessingImagesRedux.ts

```diff
@@ -1,11 +1,10 @@
 import { createAction, ActionType, getType } from 'typesafe-actions'
-import { ThreadId, SharedImage, AddResult, BlockId } from '../Models/TextileTypes'
+import {ThreadId, SharedImage, AddResult, BlockId, ILocalPhotoResult} from '../Models/TextileTypes'
 import { RootState } from './Types'
 
 const actions = {
   insertImage: createAction('processingImages/INSERT_IMAGE', (resolve) => {
-    return (uuid: string, sharedImage: SharedImage, destinationThreadId: ThreadId, comment?: string) => resolve({ uuid, sharedImage, destinationThreadId, comment })
-    return (uuid: string, sharedImage: SharedImage, destinationThreadId: ThreadId, comment?: string) => resolve({ uuid, sharedImage, destinationThreadId, comment })
+    return (uuid: string, sharedImage: SharedImage, destinationThreadId?: ThreadId, comment?: string) => resolve({ uuid, sharedImage, destinationThreadId, comment })
```

Comment:
Made `destinationThreadId` optional. In this case, the sagas for adding/sharing images will complete once the photo is added to the wallet. This is the case for photos added automatically from the camera roll.